File: modul-05/python/PRAK505-2310817310003-Mfaridsetiawan.py
def Biodata(tahunlahir, Namaku, Asal):
    tahun_sekarang = 2023
    Umur = tahun_sekarang - tahunlahir

    print("Perkenalkan Nama Saya :", Namaku)
    print("Umur saya             :", Umur)
    print("Saya Adalah Angkatan  :", tahun_sekarang)
    print("Asal Saya dari        :", Asal)

tahunLahir = int(input())
Namaku = input()
Asal = input()

Biodata(tahunLahir, Namaku, Asal)

File: modul-05/bahasa c/PRAK505-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

void Biodata(int tahunlahir, char *Namaku, char *Asal)
{
    int tahun_sekarang = 2023;
    int umur = tahun_sekarang - tahunlahir;

    printf("Perkenalkan Nama Saya: %s\n", Namaku);
    printf("Umur Saya            : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari       : %s\n", Asal);
}

int main()
{
    int tahunlahir;
    char Namaku[30], Asal[20];

    scanf(" %d", &tahunlahir);
    scanf(" %[^\n]", &Namaku);
    scanf(" %[^\n]", &Asal);

    Biodata(tahunlahir, Namaku, Asal);
    return 0;
}